packages/mobile/src/components/lineup-tile/TrackTile.tsx

```diff
@@ -72,10 +74,15 @@ export const TrackTileComponent = ({
   const dispatch = useDispatch()
   const navigation = useNavigation()
   const currentScreen = useNavigationState((state) => state.history?.[0])
+  // @ts-expect-error -- history returning unknown[]
```

Comment:
better to do as Type[]? instead of expect error?

Reply:
yeah down